python/plugins/processing/algs/qgis/Rasterize.py

```diff
@@ -57,6 +57,7 @@
 
 
 class RasterizeAlgorithm(QgisAlgorithm):
+
```

Comment:
We are using `pip install pep257` on our projects then it's D203 and D204
http://pep257.readthedocs.io/en/latest/error_codes.html

The official PEP
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings